compiler/src/iree/compiler/Dialect/Stream/Transforms/EncodeTensors.cpp

```diff
@@ -297,10 +256,11 @@ struct EncodeTensorConstantOp
     }
 
     // Dense:
-    auto resultSize = calculateElementCount(
-        op.getLoc(), alignedType, resultDims,
-        IREE::Util::getRoundedElementByteWidth(alignedType.getElementType()),
-        rewriter);
+    Value resultSize = calculateInterfaceElementCountInBytes(
+        op.getLoc(), alignedType, resultDims, rewriter);
+    if (!resultSize) {
+      return op.emitOpError("failed to calculate total byte count");
```

Comment:
add a note about why (`"non-power-of-two sub-byte element type " + alignedType` etc) so that when a user hits this we can immediately know what the issue is

Reply:
Yup, done!